### ISSUE TITLE
Screenreader Navigation

### ISSUE THREAD

**Issue by behrenle** [behrenle/Math-Explorer#46]
To improve navigation with a screen reader, the following things should be optimized:
- Focus control when switching between views
- Focus control after triggering toolbar functions
